lib/srv/sess.go

```diff
@@ -1278,7 +1279,6 @@ func (s *session) startInteractive(ctx context.Context, scx *ServerContext, p *p
 	s.io.AddReader("reader", inReader)
 	s.io.AddWriter(sessionRecorderID, utils.WriteCloserWithContext(scx.srv.Context(), s.Recorder()))
 	s.BroadcastMessage("Creating session with ID: %v", s.id)
-	s.BroadcastMessage(SessionControlsInfoBroadcast)
```

Comment:
can we transform `SessionControlsInfoBroadcast`  into a template, move it out of srv and reuse for kube access?

Reply:
Definitely, that'll keep things clean.

Reply:
Well, with all the troubles I've had viewing kube sessions, I'll just move `SessionControlsInfoBroadcast` over to `kube/proxy/sess.go` for now. Let me know what you think @tigrato.

Reply:
After chatting w/ @tigrato I'm going to rework so SSH and K8s has the same experience.